### ISSUE TITLE
Copy error when adding files to an existing rocrate 

### ISSUE THREAD

**Issue by cedricdcc** [ResearchObject/ro-crate-py#86]
Problem: Trying to add a file to an existing ro-crate results into a copy-error that keeps the rocrate from being updated.

Use-Case: 2 scientists working together on a project and making the rocrate in multiple times. (re instantiating the same rocrate multiple times.)

Solution: An internal check if a file is already present in the rocrate folder and if already present ignore the copy

**Comment by simleo** [comment]
Assuming this is the scenario:

`temp.py`:

```python
from rocrate.rocrate import ROCrate

crate = ROCrate("/tmp/crate")
crate.add_file("/etc/fstab")
crate.write("/tmp/crate")
```

```console
$ rm -rf /tmp/crate && cp -rf test/test-data/read_crate /tmp/crate
$ python temp.py
Traceback (most recent call last):
  File "temp.py", line 5, in <module>
    crate.write("/tmp/crate")
  File "/home/simleo/git/ro-crate-py/rocrate/rocrate.py", line 474, in write
    self._copy_unlisted(self.source, base_path)
  File "/home/simleo/git/ro-crate-py/rocrate/rocrate.py", line 468, in _copy_unlisted
    shutil.copyfile(source, dest)
  File "/usr/lib/python3.6/shutil.py", line 104, in copyfile
    raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
shutil.SameFileError: PosixPath('/tmp/crate/ro-crate-metadata.jsonld') and PosixPath('/tmp/crate/ro-crate-metadata.jsonld') are the same file
```

Note that adding the file is not required to trigger the error: just trying to write the crate back to the source dir is enough.

**Comment by pauldg** [comment]
I suggested a PR for this here: [https://github.com/ResearchObject/ro-crate-py/pull/88](https://github.com/ResearchObject/ro-crate-py/pull/88)

**Comment by simleo** [comment]
Fixed in #88
